docs/sources/enterprise/access-control/permissions.md

```diff
@@ -32,7 +32,8 @@ roles:delete | permissions:delegate | Allows to delete a custom role.
 roles.builtin:list | roles:* | Allows to list built-in role assignments.
 roles.builtin:add | permissions:delegate | Allows to create a built-in role assignment.
 roles.builtin:remove | permissions:delegate | Allows to delete a built-in role assignment.
-reports.admin:write | reports:* | Allows to create or update reports.
+reports.admin:create | n/a | Allows to create reports.
```

Comment:
```suggestion
reports.admin:create | n/a | Allows to create reports.
```
Do we need "Allows to" in front of all these? The grammar is super odd and the actual content comes after "Allows to ".

Reply:
Is it inaccurate to say, "Create reports."?